IBPSA/Fluid/FixedResistances/PlugFlowPipe.mo

```diff
@@ -157,19 +198,38 @@ initial equation
     level = AssertionLevel.warning);
 
 equation
-  for i in 1:nPorts loop
-    connect(vol.ports[i + 1], ports_b[i])
-    annotation (Line(points={{70,20},{72,20},{72,6},{72,0},{100,0}},
-        color={0,127,255}));
-  end for;
-  connect(cor.heatPort, heatPort)
-    annotation (Line(points={{0,10},{0,10},{0,100}}, color={191,0,0}));
-
-  connect(cor.port_b, vol.ports[1])
-    annotation (Line(points={{10,0},{70,0},{70,20}}, color={0,127,255}));
 
-  connect(cor.port_a, port_a)
-    annotation (Line(points={{-10,0},{-56,0},{-100,0}}, color={0,127,255}));
+  connect(senMasFlo.m_flow,timDel. m_flow) annotation (Line(
+      points={{-40,-11},{-40,-40},{-12,-40}},
+      color={0,0,127},
+      smooth=Smooth.None));
+  connect(timDel.tauRev,heaLos_a. tau) annotation (Line(points={{11,-36},{64,
+          -36},{64,28},{-64,28},{-64,10}},
+                                      color={0,0,127}));
+  connect(timDel.tau,heaLos_b. tau) annotation (Line(points={{11,-44},{34,-44},
+          {34,18},{44,18},{44,10}}, color={0,0,127}));
+  connect(heaLos_a.port_a,senMasFlo. port_a)
+    annotation (Line(points={{-60,0},{-50,0}}, color={0,127,255}));
+  connect(del.port_a,res. port_b)
+    annotation (Line(points={{10,0},{0,0}}, color={0,127,255}));
+  connect(senMasFlo.port_b,res. port_a)
+    annotation (Line(points={{-30,0},{-20,0}}, color={0,127,255}));
+  connect(heaLos_b.port_a,del. port_b)
+    annotation (Line(points={{40,0},{30,0}}, color={0,127,255}));
+  connect(heaLos_b.heatPort, heatPort) annotation (Line(points={{50,10},{50,86},
+          {0,86},{0,100}}, color={191,0,0}));
+  connect(heaLos_a.heatPort, heatPort) annotation (Line(points={{-70,10},{-70,
+          86},{0,86},{0,100}}, color={191,0,0}));
+  connect(heaLos_a.port_b, port_a)
+    annotation (Line(points={{-80,0},{-100,0}}, color={0,127,255}));
+  if have_PipCap then
```

Comment:
Remove that statement. If `vol` is conditionally removed then the following connect clauses are also automatically removed at translation.
Only keep the `else` branch as `if not have_pipCap then`. 
Or even better, use a conditional instance of `IBPSA.Fluid.FixedResistances.LosslessPipe` instead to model that case, and remove all conditional statements from the equation section. This makes the model structure more visible.

Reply:
Done!